File: program183.java
/*
OUTPUT:

Enter the number:
6
Given number is:6
2       4       6
*/

import java.lang.*;
import java.util.*;

class Pattern
{
	private int iNo;
	
	public void Accept()
	{
		Scanner sobj=new Scanner(System.in);
		
		System.out.println("Enter the number:");
		iNo=sobj.nextInt();
		
		
	}
	
	public void Display()
	{
		System.out.println("Given number is:"+iNo);
		
	}
	
	public void DisplayPattern()
	{
		int iCnt=0;
		
		if(iNo<0)
		{
			iNo=-iNo;
		}
		
	
		for(iCnt=1;iCnt<=iNo;iCnt++)
		{
			if(iCnt%2==0)
			{
				System.out.print(iCnt+"\t");
			}
		}
		
	}
	
}

class program183
{
	public static void main(String arg[])
	{
		Pattern pobj=new Pattern();
		
		pobj.Accept();
		pobj.Display();
		pobj.DisplayPattern();
		
	}
}

File: program183.c
/*


*/
#include<stdio.h>


void Display(int iNo)
{
	static int iCnt=0;
	iCnt=iNo;
	while(iCnt>0)
	{
		printf("%d\t",iCnt);
		iCnt--;
		Display(iNo);
	}
	
	
	
}

int main()
{
	int iValue=0;
	printf("Enter the number:\n");
	scanf("%d",&iValue);
	
	Display(iValue);
	
	return 0;
}